projects/collect_h5.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, task_name: str, dir_path: str = os.path.expanduser("~/H5s/"))
         self._is_recording = False
         date_time_string = datetime.now().strftime("%m-%d_%H-%M-%S")
         filename = os.path.join(self.file_path, date_time_string)
```

Comment:
`filename = os.path.join(self.file_path, date_time_string + '.h5')` might be better, in case I want to change the filename format later.

Reply:
i think this is fine